js/main.js

```diff
@@ -1,24 +1,36 @@
 /* eslint-disable no-unused-vars */
 /* eslint-disable no-restricted-globals */
+
+class Book {
+  constructor(id, title, author) {
+    this.id = id;
+    this.title = title;
+    this.author = author;
+  }
+}
 const books = JSON.parse(localStorage.getItem('books')) || [];
 
+const removeBook = (id) => {
+  const mBooks = JSON.parse(localStorage.getItem('books'));
+  const remainingBooks = mBooks.filter((book) => book.id !== id);
+  const removedBooks = JSON.stringify(remainingBooks);
+  localStorage.setItem('books', removedBooks);
+  window.location.reload();
+};
```

Comment:
- [ ]  All your attributes and methods have to be inside the class.
- [ ] You should have class `methods` to add, `remove `books, and `get books` as well. Let's fix that

Here is an exemple of one of them:
```javascript
class Books {
  constructor() {
    this.books = [];
  }

  addBook(title, author) {
    id = generateID(); // Optional: you can use title or author as thier primary key instead of an ID in as much they check for uniqueness
    this.books = this.books.concat({ title, author, author });
    return this.books;
  }
}
```